#!/usr/bin/env python
PACKAGE = "camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Kp_p", double_t, 0, "Kp_p", 2, 0, 5)
gen.add("Kd_p", double_t, 0, "Kd_p", 14, 0, 100)
gen.add("Ki_p", double_t, 0, "Ki_p", 0.0001, 0, 5)
gen.add("Kp_r", double_t, 0, "Kp_r", 2, 0, 5)
gen.add("Kd_r", double_t, 0, "Kd_r", 14, 0, 100)
gen.add("Ki_r", double_t, 0, "Ki_r", 0.00008, 0, 5)

gen.add("maxAngle", double_t, 0, "maxAngle", 4, 0, 10)

exit(gen.generate(PACKAGE, "AttitudePublisher", "PID"))
